VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -494,4 +494,10 @@
   <data name="StringDotFormatWithDifferentAmountOfArgumentsTitle" xml:space="preserve">
     <value>Invalid arguments for the string.Format() call.</value>
   </data>
+  <data name="LoopedRandomInstantiationAnalyzerMessage" xml:space="preserve">
+    <value>Instantiate the random instance outside the loop.</value>
```

Comment:
This doesn't read like something you see in an error list. "*Variable <var> of type Random is instantiated in a loop*" seems more appropriate (note that you have to tell which variabledeclarator it is).